lua/gk.lua

```diff
@@ -48,6 +54,12 @@ return function (net_conf, lls_conf, sol_conf, gk_lcores)
 	gk_conf.back_max_pkt_burst =
 		gatekeeper.get_back_burst_config(
 			gk_max_pkt_burst_back, net_conf)
+
+	gk_conf.front_icmp_msgs_per_sec = front_icmp_msgs_per_sec
```

Comment:
Shouldn't we replace `front_icmp_msgs_per_sec` with `math.floor(front_icmp_msgs_per_sec / gk_conf.num_lcores + 0.5)`? If so, do the same for the back interface.